Source/Core/Core/AchievementManager.h

```diff
@@ -137,6 +137,7 @@ class AchievementManager
 
   std::recursive_mutex& GetLock();
   bool IsHardcoreModeActive() const;
+  void SetHardcoreMode();
```

Comment:
I don't see why the GUI should call this. It's already being called when it needs to be, thanks to the config callback that's registered by AchievementManager.cpp.